fpsim/locations/senegal/data/contra_coef_mid.csv

```diff
@@ -0,0 +1,14 @@
+"Estimate","Std. Error","t value","Pr(>|t|)","rhs"
+-20.6446422859391,0.156922447180107,-131.559522916721,4.03408441480223e-176,"(Intercept)"
+-0.414945894181982,0.199725532444401,-2.07758061327234,0.0392165008804567,"ns(age, knots = c(25, 40))1"
+-0.814251406002529,0.26621182886774,-3.05865975026627,0.0025749923686252,"ns(age, knots = c(25, 40))2"
+-0.5970591862295,0.187715760255941,-3.18065561152372,0.00174008083612002,"ns(age, knots = c(25, 40))3"
+22.277008770357,0.350596417836769,63.5403205423758,2.47186942579002e-122,"fp_ever_user"
+0.192957931602284,0.149393423494167,1.29160927629333,0.198205550872494,"edu.levelPrimary"
+0.453261226921791,0.174643333207947,2.59535373378437,0.0102561867903628,"edu.levelSecondary"
+0.166464022607253,0.0347139015325769,4.7953129800474,3.47229640719587e-06,"parity"
+-0.112718298851514,0.133063440077691,-0.847101944649123,0.398101922544714,"urban"
+-0.0666872316676885,0.0552611650201476,-1.20676485273836,0.229160782232243,"wealthquintile"
```

Comment:
Note that urban and wealth have the wrong sign

Reply:
Not wrong, just opposite haha! But these are both not statistically significant, so once we are including uncertainty around these estimates they should basically be 0.